site/public/zipkin.proto

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2018-2024 The OpenZipkin Authors
```

Comment:
also I don't know why we are pushing this and the thrift to the swagger site, as they haven't anything to do with swagger. Maybe simplest to bring this PR back to swagger, as that's only about zipkin-api.yaml.

When we do much more than requested, we end up with more decisions to make and that slows down change. Let's focus on the problem which was only about swagger and only about XSS. Theres' no relevance to the proto and thrift, so let's keep it out of this so we can land the fix?